client/public/style.css

```diff
@@ -0,0 +1,19 @@
+.headList{
+    list-style-type:none;
+    margin:0;
+    padding:0;
+}
+.headOption{
+      margin:10px;
+      display:inline-block;
+      margin-left:10px;
+}
+
+.headLink{
+    text-decoration:none;
+    position:relative;
+}
+.headLink:hover {
+    text-decoration:underline;
+    color:#555
+}
```

Comment:
Make sure your editor is adding a final new line when you save.

If you're using VSCode, put this in your settings file:
`"files.insertFinalNewline": true,`

Usually not a big deal, but it can break certain things (like cat on linux) since the POSIX standard defines a line as some sequence of bytes followed by a newline character, and apps are written with that in mind.

Reply:
ok thanks for that, I changed my settings and updated branch for the final new line.